.gitignore

```diff
@@ -3,6 +3,7 @@
 *.py[cod]
 *.so
 *.cfg
+!docker/actinia-plugin-tests/actinia-test.cfg
```

Comment:
Is this not used?

Reply:
It is used.

Reply:
But when it is used, why is it ignored?

Reply:
I thought the `!` would not irgnore it again because all `cfg` are ignored

Reply:
:see_no_evil: upsi sorry.